index.html

```diff
@@ -107,154 +114,161 @@ <h2 class="work_primary_text">Multi-Post Stories</h2>
+                            industry's standard.
+                        </p>
+                        <div class="tags">
+                            <ul class="tag">
+                                <li> <a href="#">HTML</a></li>
+                                <li> <a href="#">Bootstrap</a></li>
+                                <li> <a href="#">Ruby</a></li>
+                            </ul>
+                        </div>
 
-                    <div class="btn2-holder">
-                        See Project
+                        <div class="btn2-holder">
+                            See Project
+                        </div>
                     </div>
                 </div>
-            </div>
-            <div class="grid-item item2">
-                <div class="content">
-                    <h3 class="p-text">Professional Art <br>
-                        Printing Data
-                    </h3>
-                    <p class="s-text">
-                        A daily selection of privately
-                        personalized reads; no accounts or
-                        sign-ups required. Has been the 
-                        industry's standard.
-                    </p>
-                    <div class="tags">
-                        <ul class="tag">
-                            <li> <a href="#">HTML</a></li>
-                            <li> <a href="#">Bootstrap</a></li>
-                            <li> <a href="#">Ruby</a></li>
-                        </ul>
-                    </div>
+                <div class="grid-item item2">
+                    <div class="content">
+                        <h3 class="p-text">Professional Art <br>
+                            Printing Data
+                        </h3>
+                        <p class="s-text">
+                            A daily selection of privately
+                            personalized reads; no accounts or
+                            sign-ups required. Has been the 
+                            industry's standard.
+                        </p>
+                        <div class="tags">
+                            <ul class="tag">
+                                <li> <a href="#">HTML</a></li>
+                                <li> <a href="#">Bootstrap</a></li>
+                                <li> <a href="#">Ruby</a></li>
+                            </ul>
+                        </div>
 
-                    <div class="btn2-holder">
-                        See Project
+                        <div class="btn2-holder">
+                            See Project
+                        </div>
                     </div>
                 </div>
-            </div>
+        </section>
         </div>
         <section class="main-container">
             <article class="about-container">
                 <article class="about">
+                    <article class="about-top">
                         <h2 class="about-p-text" >About Me</h2>
```

Comment:
- [ ] Kindly use **h4** instead of **h2** because it optimizes the main heading of your portfolio website and enhances the user experience. The `h2` tag is specifically designed for secondary headings which you already used in the `hero-section` and actually provides a clear and prominent representation of the page's secondary topic or purpose. By utilizing the *h4* tag, you create a visual hierarchy that helps users quickly identify and determine the relevance of the content to their needs and interests. This improves the overall usability and navigation of your website, ensuring a more engaging and effective experience for visitors.